treee111/wahooMapsCreator
# [FEAT] Read tag-wahoo.xml and tags-to-keep.json from user directory first (from python module as fallback)

## This PR…

- reads `tag-wahoo.xml` (or with another name, see argument `-tag`) and `tags-to-keep.json` from user directory if they exist, if not read from python module
- introduces a function to access config files from user directory `wahooMapsCreatorData/_config` therefore
- introduces a new cli call `python -m wahoomc.init` for initialization of the user directory. Files from python module are copied to user directory and you are asked for overwriting if they aready exist
- updates documentation accordingly

## Considerations and implementations

Lately a lot of people played around with tags-to-keep and tag-wahoo .xml files.
To do that one had to 1. clone/download the repo or 2.  make changes in the python module installation controlled by PyPI.
The advice is now more than ever to install via PyPI and put user specific files in the unser directory of wahooMapsCreator!

## How to test

1. initialize via wahoomc.init
2. run for your country or X/Y
3. adjust a file in the user directory
4. run again, check if the change is visible in logging or the generated maps

## Pull Request Checklist
    + Especially the [Pull-Requests](https://github.com/treee111/wahooMapsCreator/blob/develop/.github/CONTRIBUTING.md#Pull-Requests) section
